content/en/llm_observability/configuration/_index.md

```diff
@@ -0,0 +1,51 @@
+--- 
+title: Configuration
+description: Learn how to configure topics and evaluations for your LLM applications on the Configuration page.
+further_reading: 
+- link: "/llm_observability/terms/" 
+  tag: "Documentation" 
+  text: "Learn about Evaluations"
+- link: "/llm_observability/submit_evaluations/" 
+  tag: "Documentation" 
+  text: "Learn how to submit Evaluations"
+- link: "/llm_observability/setup" 
+  tag: "Documentation" 
+  text: "Learn how to set up LLM Observability"
+---
+
+## Overview
+
+You can configure your LLM applications on the [Configuration page][1] with settings that can optimize your application’s performance and security. 
+
+Topics
+: Helps identify and filter out irrelevant or malicious input, ensuring your LLM application stays focused on its intended purpose. 
+
+Evaluations
+: Enables Datadog to assess your LLM application against respective dimensions like Quality and Security & Safety. By enabling evaluations, you can assess the effectiveness of your application’s responses and maintain high standards for both performance and user safety. 
```

Comment:
```suggestion
: Enables Datadog to assess your LLM application against respective dimensions like Quality and Security and Safety. By enabling evaluations, you can assess the effectiveness of your application’s responses and maintain high standards for both performance and user safety. 
```
Because it's "Security and Safety" elsewhere